### ISSUE TITLE
how to place the dataset and train the model, i have many errors

### ISSUE THREAD

**Issue by sadok11** [sanghyun-son/EDSR-PyTorch#307]
![Capture d’écran du 2021-05-07 19-44-36](https://user-images.githubusercontent.com/38444076/117488988-535f9080-af6d-11eb-9651-27832d7a9fee.png)

**Comment by wusuhuang** [comment]
i have the same question

**Comment by cyinen** [comment]
> i have the same question

Have you solved it? I met the same problem

**Comment by wwang0107** [comment]
> 

Turn the shuffle(in dataloader) to False
